rules/windows/persistence_time_provider_mod.toml

```diff
@@ -13,7 +13,7 @@ abuse this architecture to establish persistence, specifically by registering an
 provider.
 """
 from = "now-9m"
-index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+index = ["logs-endpoint.events.*"]
```

Comment:
The integration is only parsing DWORD and QWORD registry values, so registry.data.strings will always be null here. Dropping Support